mix.exs

```diff
@@ -15,7 +15,7 @@ defmodule Demo.Mixfile do
   # Type "mix help compile.app" for more information
   def application do
     # Specify extra applications you'll use from Erlang/Elixir
-    [extra_applications: [:logger, :cowboy, :phoenix_pubsub],
+    [extra_applications: [:logger, :cowboy, :phoenix_pubsub, :httpoison],
```

Comment:
Since 1.4 Applications from dependencies are automatically started, so I don't think this should be needed normally